src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -728,6 +734,76 @@ private void createGitHubRepository(Channel channel, User sender, String name, S
         }
     }
 
+    /**
+     * Makes GitHub team visible.
+     *
+     * @param team
+     *      team to make visible
+     */
+    private boolean makeGitHubTeamVisible(Channel channel, User sender, String team) {
+        boolean result = false;
+        if (!isSenderAuthorized(channel, sender)) {
+            insufficientPermissionError(channel);
+            return false;
+        }
+        OutputChannel out = channel.send();
+        try {
+            GitHub github = GitHub.connect();
+            GHOrganization o = github.getOrganization(IrcBotConfig.GITHUB_ORGANIZATION);
+
+            final GHTeam ghTeam = o.getTeamByName(team);
+            if (ghTeam == null) {
+                out.message("No team for " + team);
+                return false;
+            }
+
+            ghTeam.setPrivacy(GHTeam.Privacy.CLOSED);
+
+            String successMsg = "Made GitHub team " + team + " visible";
+            out.message(successMsg);
+            result = true;
+        } catch (IOException e) {
+            out.message("Failed to make GitHub team " + team + " visible: " + e.getMessage());
+            e.printStackTrace();
+        }
+        return result;
+    }
+
+    /**
+     * Makes a user a maintainer of a GitHub team
+     *
+     * @param team
+     *      make user a maintainer of a team.
+     */
+    private boolean makeGitHubTeamMaintainer(Channel channel, User sender, String newTeamMaintainer, String team) {
+        boolean result = false;
+        if (!isSenderAuthorized(channel, sender)) {
+            insufficientPermissionError(channel);
+            return false;
+        }
+        OutputChannel out = channel.send();
+        try {
+            GitHub github = GitHub.connect();
+            GHUser c = github.getUser(newTeamMaintainer);
+            GHOrganization o = github.getOrganization(IrcBotConfig.GITHUB_ORGANIZATION);
+
+            final GHTeam ghTeam = o.getTeamByName(team);
+            if (ghTeam == null) {
+                out.message("No team for " + team);
+                return false;
+            }
+
+            ghTeam.add(c, GHTeam.Role.MAINTAINER);
+            String successMsg = "Added " + newTeamMaintainer + " as a GitHub maintainer for team " + team;
```

Comment:
Same as above, not sure why this would need to be a separate declaration instead of just passing the string to `out.message`